customer-addresses.go

```diff
@@ -2,6 +2,45 @@ package shopify
 
 // CustomerAddressRepository maintains the customer addresses of a shop.
 type CustomerAddressRepository interface {
+	// Create creates a new address against the specified customer
+	Create(id int64, address CustomerAddress) (CustomerAddress, error)
 	// Delete deletes the specified address for the specified customer
 	Delete(id int64, addressID int64) error
 }
+
+type CustomerAddress struct {
+	// Address1 - the street address
+	Address1 string
+	// Address2 - an optional additional field for the street address
+	Address2 string
+	// City - the city, town, or village
+	City string
+	// Company - the company of the person associated with this address
+	Company string
+	// Country - the name of the country
+	Country string
+	// CountryCode - the two-letter code (ISO 3166-1 format) for the country of the address
+	CountryCode string
+	// CountryName - the name of the country
+	CountryName string
+	// CustomerID - the ID of the customer which address is stored against.
+	CustomerID uint64
+	// Default - Whether this is the customer default address.
+	Default bool
```

Comment:
The Customer Address resource does not have this field from what I can see in the docs 🤔

Reply:
I'm pretty sure it's there ([here](https://shopify.dev/api/admin-rest/2022-10/resources/customer-address#get-customers-customer-id-addresses?limit=1)) 🤔

Reply:
Oh.. I was looking at the `The Customer Address resource` section and is not there.. I guess they should have added it right?

Reply:
Yeah not sure why it's not there tbh 😅 but since the `GET` request is returning that field, I'd rather leave it there 🙂